shell/config/product/explorer.js

```diff
@@ -233,8 +235,11 @@ export function init(store) {
       value:  'metadata.fields.1',
       sort:   'metadata.fields.1',
       search: 'metadata.fields.1',
+    }, {
+      ...SECRET_DATA,
+      sort:   false,
```

Comment:
bug fix. this is a locally computed property so can't be sorted or filtered remotely